core/hydromet_distal.py

```diff
@@ -0,0 +1,75 @@
+from hydromet import*
+
+
+#---------------------------------------------------------------------------#
+def main(md: dict, dur: str, name: str, CN: int, arc_data: dict, AOI: str, 
+                            outputs_dir: str, adjust_CN_less24: bool=False, 
+                    remove_intermediates: bool=True, display_print: bool=True, 
```

Comment:
lets add a description list for arguments for the main functions.

example:

```python
def main(md: dict, dur: str, name: str, CN: int, arc_data: dict, AOI: str, 
                            outputs_dir: str, adjust_CN_less24: bool=False, 
                    remove_intermediates: bool=True, display_print: bool=True, 
                                                    plot: bool=True) -> list:
    '''Extracts data from the metadata dictionary, calculates random curve 
       numbers, performs the excess rainfall calculation, groups the events,
       saves the grouped incremental excess rainfall and metadata, and plots
       the results.


    Parameters
    ----------
    md: markdown dictionary of metadata generated in ...
    dur: ...etc
    name: 
    CN: 
    arc_data: 
    AOI: 
    outputs_dir: 
    adjust_CN_less24:
    remove_intermediates:
    display_print:
    plot: 

    Returns
    -------
    List of ...

    '''

```

Reply:
Issue resolved with 512c161993a047eb82ac431f5502a0f2a2e57ca9.